store/interfaces.json

```diff
@@ -48,6 +53,9 @@
   "op-order-number": {
     "component": "OrderNumber"
   },
+  "op-order-billing": {
+    "component": "OrderBilling"
+  },
```

Comment:
Remove this block because it doesn't suit every store

Reply:
Removed everything related billing from order-placed app and moved it to a custom app instead.